src/Decorators/property.ts

```diff
@@ -23,15 +22,15 @@ function annotate(
   target: any,
   propertyName: string,
   propertyDescriptor?: PropertyDescriptor,
-  optionsOrType: Partial<ModelPropertyDecoratorOptions> | DataType = {},
+  options: Partial<ModelPropertyDecoratorOptions> = {},
 ): void {
-  const options: Partial<ModelPropertyDecoratorOptions> = {
-    ...(optionsOrType as ModelPropertyDecoratorOptions),
+  const parsedOptions: Partial<ModelPropertyDecoratorOptions> = {
+    ...(options as ModelPropertyDecoratorOptions),
```

Comment:
This casting should no longer be needed, right?

Reply:
I guess not, no.
Removing...